extensions/2.0/Vendor/EXT_texture_avif/README.md

```diff
@@ -0,0 +1,109 @@
+# EXT_texture_avif
+
+## Contributors
+
+- Omar Shehata, Cesium, [@OmarShehata](https://github.com/OmarShehata)
+- Pascal Massimino, Google, [@Skal65535](https://github.com/skal65535)
+
+## Status
+
+Complete
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension allows glTF models to use AVIF as a valid image format. A client that does not implement this extension can ignore the provided AVIF image and continue to rely on the PNG and JPG textures available in the base specification. Defining a fallback texture is optional. The [best practices](#best-practices) section describes the intended use case of this extension and the expected behavior when using it without a fallback texture.
+
+AVIF is an image format developed by Netflix that uses the AV1 VideoCodec [specification](https://aomediacodec.github.io/av1-avif/).
+The format has gained popularity and you can see in what browsers it is supported here [caniuse](https://caniuse.com/avif)
+
+## glTF Schema Updates
+
+The `EXT_texture_avif` extension is added to the `textures` node and specifies a `source` property that points to the index of the `images` node which in turn points to the AVIF image.
+
+The following glTF will load `image.avif` in clients that support this extension, and otherwise fall back to `image.png`.
+
+```
+"textures": [
+    {
+        "source": 0,
+        "extensions": {
+            "EXT_texture_avif": {
+                "source": 1
+            }
+        }
+    }
+],
+"images": [
+    {
+        "uri": "image.png"
+    },
+    {
+        "uri": "image.avif"
+    }
+]
+```
+
+When used in the glTF Binary (.glb) format the `images` node that points to the AVIF image uses the `mimeType` value of `image/avif`.
+
+```
+"textures": [
+    {
+        "source": 0,
+        "extensions": {
+            "EXT_texture_avif": {
+                "source": 1
+            }
+        }
+    }
+],
+"images": [
+    {
+        "mimeType": "image/png",
+        "bufferView": 1
+    },
+    {
+        "mimeType": "image/avif",
+        "bufferView": 2
+    }
+]
+```
+
+### JSON Schema
+
+[glTF.EXT_texture_avif.schema.json](schema/glTF.EXT_texture_avif.schema.json)
+
+## Best Practices
+
+Since AVIF is not as widely supported as JPEG and PNG, it is recommended to use this extension only when transmission size is a significant bottleneck. For example, in geospatial applications the total texture payload can range from gigabytes to terabytes of data. In those situations, AVIF textures can be used without a fallback to improve storage and transmission.
```

Comment:
Replace the copy-pasted WebP rationale with something specific to AVIF. The section should mention concrete glTF use-cases that benefit from using this extension.

Reply:
I changed it a bit, but copy is now my strong suite, feel free to update it to what you think is better.

For me the main advantage while developing mobile open gl is transfer size on 4g.
beeing able to only have to download 2mb instead of 20 is a big win.

Reply:
That's only a win on network transmission / disk storage. AVIF decoding complexity is higher than with other formats and GPU memory usage is the same as JPEG / PNG / WebP. For example (don't copy this literally), a 3D application that features real photos / paintings may prioritize transmission size over other other metrics.

In a case of glTF, images are used as textures, for which GPU throughput is usually more important, so our general recommendation is to use `KHR_texture_basisu`.

This section should provide glTF use-cases that prioritize transmission / storage size and are not bound by GPU memory or throughput.